great-idea/css/index.css

```diff
@@ -63,4 +63,111 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	/* border: 1px solid red; */
+	margin: 0 auto; 
+	width: 900px;
+}
+
+header {
+	/* border: 3px dotted black; */
+	display: flex;
+}
+
+header nav {
+	text-decoration: none;
+	/* border: 1px solid blue; */
+	color: grey;
+	width: 600px;
+	height: 50px;
+	align-items: center; 
+	display: flex;
+	justify-content: space-evenly;
+	
+}
+
+header nav a {
+	text-decoration: none;
+	font-size: 14px;
+	color: grey;
+	
+}
+
+header .logo {
+    /* border: 1px solid blue; */
+	margin-left: 100px;
+	display: flex;
```

Comment:
flex on the image itself won't do anything for you. remember it goes on the parent element.